src/processbuildevents.py

```diff
@@ -22,9 +22,6 @@ def handler(build_event, context):
     LOG.debug('Received event: %s', build_event)
 
     build = Build(build_event)
-    if build.project_name != config.PROJECT_NAME:
-        LOG.debug('Not our codebuild project')
-        return
```

Comment:
nit: Removing this check introduces small race condition for users who had a previous version of this app installed and upgrade to a version containing this change. Since the CloudWatch event rule depends on the Lambda function, CloudFormation will update the Lambda function before updating the event rule. That means there will be a brief time period where the Lambda function is not performing a check on the project name, but the CloudWatch event rule filter has not yet been updated to filter out events from other build projects.

However, in the case of this app, I think the likelihood of anyone hitting this is extremely small and even if they did, the impact is not high (extra PR comment), so I think we can take this as is.